modules/core/src/test/scala/org/scalasteward/core/repoconfig/RepoConfigAlgTest.scala

```diff
@@ -26,6 +26,7 @@ class RepoConfigAlgTest extends AnyFunSuite with Matchers {
          |updates.ignore = [ { groupId = "org.acme", version = "1.0" } ]
          |updates.limit = 4
          |updates.includeScala = true
+         |updates.fileExtensions = [ "*.txt" ]
```

Comment:
I know this is just a test, but maybe we should remove `*` here to not give the anyone the impression that it is needed for `fileExtensions`.

Reply:
I agree. It would be a bad example.